web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -38,118 +38,185 @@ function tabs(elem, indirect) {
 }
 
 
-$(function(){
-    var $html = $('html');
-
-    var cobrand = $('meta[name="cobrand"]').attr('content');
-
-    if (typeof variation !== 'undefined' && variation === 1) {
-        $('input[name=variant]').val(1);
-    }
+(function($){
 
-    // Deal with switching between mobile and desktop versions on resize
-    var last_type;
-    $(window).resize(function(){
-        var type = Modernizr.mq('(min-width: 48em)') || $('html.iel8').length ? 'desktop' : 'mobile';
-        if (last_type == type) { return; }
-        if (type == 'mobile') {
-            $html.addClass('mobile');
-            $('#map_box').css({ height: '10em' });
-            if (typeof fixmystreet !== 'undefined') {
-                fixmystreet.state_map = ''; // XXX
+    // A sliding drawer from the bottom of the page, small version
+    // that doesn't change the main content at all.
+    $.fn.small_drawer = function(id) {
+        var opened, $this = $(this), d = $('#' + id);
```

Comment:
There is a reason `opened` was outside this function, and it's so it was a shared variable between the small drawers (so that clicking one when there's one open would close the open one). Moving it inside breaks this.